parser-typechecker/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -2221,3 +2266,13 @@ executePPE unisonFile =
     makeShadowedPrintNamesFromLabeled
       (UF.termSignatureExternalLabeledDependencies unisonFile)
       (UF.typecheckedToNames0 unisonFile)
+
+loadTypeOfTerm :: Referent -> Action m i v (Maybe (Type v Ann))
+loadTypeOfTerm (Referent.Ref r) = eval $ LoadTypeOfTerm r 
+loadTypeOfTerm (Referent.Con (Reference.DerivedId r) cid _) = do
+  decl <- eval $ LoadType r
+  case decl of
+    Just (either DD.toDataDecl id -> dd) -> pure $ DD.typeOfConstructor dd cid
+    Nothing -> pure Nothing
+loadTypeOfTerm (Referent.Con r cid _) = error $
+  reportBug "924628772" "Attempt to load a type declaration which is a builtin!"
```

Comment:
Where does this bugId come from? Is it just arbitrarily selected?

Reply:
Right